pure_pursuit_jerk_wrapper/src/pure_pursuit_jerk_wrapper_node.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2018-2020 LEIDOS.
```

Comment:
2021

Reply:
Please double check all the copyrights

Reply:
Should this be 2020-2021?

Reply:
Just 2021 since that is the year the file was created